pkg/roachprod/install/session.go

```diff
@@ -331,10 +331,9 @@ var sshAuthArgsOnce sync.Once
 
 func sshAuthArgs() []string {
 	sshAuthArgsOnce.Do(func() {
-		paths := []string{
-			filepath.Join(config.OSUser.HomeDir, ".ssh", "id_ed25519"),
-			filepath.Join(config.OSUser.HomeDir, ".ssh", "id_rsa"),
-			filepath.Join(config.OSUser.HomeDir, ".ssh", "google_compute_engine"),
+		paths := make([]string, len(config.DefaultPubKeyNames))
+		for idx, name := range config.DefaultPubKeyNames {
+			paths[idx] = filepath.Join(config.OSUser.HomeDir, ".ssh", name)
```

Comment:
Nit: `filepath.Join(config.SSHDirectory, name)`.

Reply:
Ah good catch, will update before merging.

Reply:
🤦 somehow managed to half-fix this right before merging
fixed in: https://github.com/cockroachdb/cockroach/pull/125214